sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 ### Added
 - Added CommunicationIdentityClient (originally was part of the azure.communication.administration package).
+- Added ability to create a user and issue token for it at the same time.
+
+### Breaking
```

Comment:
also, is this naming from the C# PR? Looks like Pascal Case

Reply:
Yeah, I'll update this.